### PR TITLE
New Cute Compliment

### PR DESCRIPTION
**Pull request by Rai99**
Issue: #2786 
Almost overlapped by: #2818

I thought this was a bit too similar to other compliments, but a cute compliment was wanted, so I finished it. 

Hopefully affection inflation isn't a problem yet.

### CODE REVIEW

**Comment by AsuraShun** [comment]
this is making me excited

File: Monika After Story/game/script-compliments.rpy [hunk 1]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+label mas_compliment_cute_2:
+    m 1wubfsdld "Ah!"
+    m 3rkbsa "You {i}kind of{/i} caught me off guard with that one."
+    m "Just a little..."
+    m 1hubfa "But I'm glad you think so!"
+    menu:
+        "Seeing you always warms my heart!":
+            $ mas_gainAffection(5,bypass=True)
+            m 1hubfb "Aww, it really warms my heart to hear you say that!"
+            m 1dkbfa "Just wait until I finally get to hug you for real!"
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 1dkbfu "Just wait until I finally get to hug you for real!"
```

File: Monika After Story/game/script-compliments.rpy [hunk 2]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+            m 1ekbfa "I can barely contain myself just thinking about it, ehehe~"
+        "You're even cuter when you're flustered":
+            $ mas_gainAffection(3,bypass=True)
+            m 1tku "Not letting it go, huh, [player]?"
+            m 3tsbsa "Hmph, I just wasn't expecting it."
+            m 3tubfb "Don't expect it to be so easy next time, [player]."
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 3tubfb "Don't expect it to be so easy next time..."
```

File: Monika After Story/game/script-compliments.rpy [hunk 3]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+            m 1tsbsa "I'll get you back someday, eheheh~"
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 1tubfu "I'll get you back someday, ehehe~"
```

File: Monika After Story/game/script-compliments.rpy [hunk 4]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+    m 3rkbsa "You {i}kind of{/i} caught me off guard with that one."
+    m "Just a little..."
+    m 1hubfa "But I'm glad you think so!"
+    menu:
+        "Seeing you always warms my heart!":
+            $ mas_gainAffection(5,bypass=True)
+            m 1hubfb "Aww, it really warms my heart to hear you say that!"
+            m 1dkbfa "Just wait until I finally get to hug you for real!"
+            m 1ekbfa "I can barely contain myself just thinking about it, ehehe~"
+        "You're even cuter when you're flustered":
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
        "You're even cuter when you're flustered.":
```

File: Monika After Story/game/script-compliments.rpy [hunk 5]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+            $ mas_gainAffection(3,bypass=True)
+            m 1tku "Not letting it go, huh, [player]?"
+            m 3tsbsa "Hmph, I just wasn't expecting it."
+            m 3tubfb "Don't expect it to be so easy next time, [player]."
+            m 1tsbsa "I'll get you back someday, eheheh~"
+        "You're as cute as Natsuki":
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
        "You're as cute as Natsuki.":
```

File: Monika After Story/game/script-compliments.rpy [hunk 6]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+            $ mas_loseAffection(modifier=0.5)
+            m 2lfc "Oh..."
+            m 1rsc "Thanks [player], but I was hoping I would stand in my own category."
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 1rsc "But I was kind of hoping I would stand in my own category."
```

File: Monika After Story/game/script-compliments.rpy [hunk 7]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+label mas_compliment_cute:
+    if not renpy.seen_label("mas_compliment_cute_2"):
+        call mas_compliment_cute_2
+    else:
+        call mas_compliment_cute_3
+    return
+
+label mas_compliment_cute_2:
+    m 1wubfsdld "Ah!"
+    m 3rkbsa "You {i}kind of{/i} caught me off guard with that one."
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
    m 3rkbfsdla "You {i}kind of{/i} caught me off guard with that one."
```

File: Monika After Story/game/script-compliments.rpy [hunk 8]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+    m "Just a little..."
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
    m 3tubfb "Just a little..."
```

File: Monika After Story/game/script-compliments.rpy [hunk 9]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+    m 1hubfa "But I'm glad you think so!"
+    menu:
+        "Seeing you always warms my heart!":
+            $ mas_gainAffection(5,bypass=True)
+            m 1hubfb "Aww, it really warms my heart to hear you say that!"
+            m 1dkbfa "Just wait until I finally get to hug you for real!"
+            m 1ekbfa "I can barely contain myself just thinking about it, ehehe~"
+        "You're even cuter when you're flustered":
+            $ mas_gainAffection(3,bypass=True)
+            m 1tku "Not letting it go, huh, [player]?"
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 2tubfu "Not letting it go, huh, [player]?"
```

File: Monika After Story/game/script-compliments.rpy [hunk 10]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+            m 3tsbsa "Hmph, I just wasn't expecting it."
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 2rubfu "Hmph, I just wasn't expecting it."
```

File: Monika After Story/game/script-compliments.rpy [hunk 11]
```diff
@@ -446,6 +446,55 @@ init 5 python:
...
+            m 3tubfb "Don't expect it to be so easy next time, [player]."
+            m 1tsbsa "I'll get you back someday, eheheh~"
+        "You're as cute as Natsuki":
+            $ mas_loseAffection(modifier=0.5)
+            m 2lfc "Oh..."
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 2lfc "Oh. {w=1}Thanks, [player]..."
```

File: Monika After Story/game/script-compliments.rpy [hunk 12]
```diff
@@ -446,6 +446,55 @@ init 5 python:
         eventdb=store.mas_compliments.compliment_database
     )
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_compliments_database,
+            eventlabel="mas_compliment_cute",
+            prompt="... You're cute!",
```
Comment (multimokia) [inline Monika After Story/game/script-compliments.rpy]: New prompt format for compliments
```suggestion
            prompt="You're cute!",
```

File: Monika After Story/game/script-compliments.rpy [hunk 13]
```diff
@@ -477,6 +477,55 @@ init 5 python:
...
+            m 2lfc "Oh. {w=1}Thanks, [player]..."
+            m 1rsc "But I was kind of hoping I would stand in my own category."
+    return
+
+label mas_compliment_cute_3:
+    python:
+        thanks_quip = renpy.random.choice(store.mas_compliments.thanking_quips)
+        thanks_quip = renpy.substitute(thanks_quip)
+    m 1ekbfa "[thanks_quip]"
+    m 1hubfa "You can be really cute too a lot of the time, you know!"
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
    m 1hubfa "You can be really cute a lot of the time too, you know~"
```

File: Monika After Story/game/script-compliments.rpy [hunk 14]
```diff
@@ -477,6 +477,55 @@ init 5 python:
...
+label mas_compliment_cute_2:
+    m 1wubfsdld "Ah!"
+    m 3rkbfsdla "You {i}kind of{/i} caught me off guard with that one."
+    m 3tubfb "Just a little..."
+    m 1hubfa "But I'm glad you think so!"
+    menu:
+        "Seeing you always warms my heart!":
+            $ mas_gainAffection(5,bypass=True)
+            m 1hubfb "Aww, it really warms my heart to hear you say that!"
+            m 1dkbfu "Just wait until I finally get to hug you for real!"
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 1dkbfu "...Almost as much as when I picture us finally being together in the same reality."
```

File: Monika After Story/game/script-compliments.rpy [hunk 15]
```diff
@@ -477,6 +477,55 @@ init 5 python:
...
+            m 1ekbfa "I can barely contain myself just thinking about it, ehehe~"
```
Comment (jmwall24) [inline Monika After Story/game/script-compliments.rpy]:
```suggestion
            m 1ekbfa "I can barely contain myself just imagining that special day~"
```

**Comment by ThePotatoGuy** [comment]
this is ripe for pm vars. I will add this now

**Pull request merged**